pkg/operator/api/api.go

```diff
@@ -129,6 +134,14 @@ func (a *apiServer) OnComponentUpdated(_ context.Context, component *componentsa
 	a.connLock.Unlock()
 }
 
+func (a *apiServer) OnHTTPEndpointUpdated(_ context.Context, endpoint *httpendpointsapi.HTTPEndpoint) {
+	a.endpointLock.Lock()
+	for _, endpointUpdateChan := range a.allEndpointsUpdateChan {
+		endpointUpdateChan <- endpoint
```

Comment:
This should probably happen within a background goroutine, i.e:

```go
go func(endpointUpdateChan chan *httpendpointsapi.HTTPEndpoint) {
    endpointUpdateChan <- endpoint
}(endpointUpdateChan)
```

Reason is that otherwise if one of the consumer is busy/hanging, it will block every other consumer from receiving the updated state. Yes, the channel is buffered, but only with capacity of 1, so a second message would block if the first one isn't consumed.